src/pages/doctor/profile/styles.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export const PageContainer = styled.div`
+  display: flex;
+`;
```

Comment:
where is empty line, I think that prettier not configured

Reply:
These are not my changes, this code is from the pool from the feat/edit-profile branch, which was made by @DaniilVolobuiev - so, it was his task.